robotoff/insights/question.py

```diff
@@ -164,6 +170,47 @@ def get_source_image_url(barcode: str) -> Optional[str]:
 
         return None
 
+    @staticmethod
+    def generate_selected_images(images: JSONType, barcode: str) -> JSONType:
+        selected_images = {}
+        images_data = {}
```

Comment:
it's more concise to construct it this way:
`select_images = {"front": {}, "small": {}, "thumb": {}, "display": {}}`